### ISSUE TITLE
[BUG] Elevation import does not work with rampTime = 0

### ISSUE THREAD

**Issue by jtgrasb** [WEC-Sim/WEC-Sim#916]
**Note: italicized text below is include as an example and should be updated before submission. If you feel any section is not applicable to your request, please replace with 'N/A' rather than delete a section.**

**Describe the bug**
The elevation import method for the waveClass gives the following error when the rampTime equals zero.

![image](https://user-images.githubusercontent.com/87095491/180259146-fdb1b625-0bcf-4647-a8be-6526af09afda.png)

**WEC-Sim file(s)**
Seems to be related to the way the elevation file is read in within the waveClass

**To Reproduce**
_Steps to reproduce the behavior:_
1. Go to RM3 example
2. Change waves to elevation import and rampTime to 0.
3. Run wecSim.

**Expected behavior**
WEC-Sim run without error.

**Workstation Information (this section MUST BE completed):**
 - OS: Windows 10
 - MATLAB 2021b
 - WEC-Sim v5.0

**Additional context**

**Comment by kmruehl** [comment]
Close this issue once #917  is merged

**Comment by nathanmtom** [comment]
Closing this issue as #917 has been merged into the the `dev` branch of WEC-Sim.
